### PR TITLE
🌲 use log level set by environment and service

### PR DESCRIPTION
**Pull request by idrop**
the log_level by environment is [set in this verify-infrastructure-config PR](https://github.com/alphagov/verify-infrastructure-config/pull/325).

### CODE REVIEW

File: terraform/modules/hub/files/tasks/hub-policy.json [hunk 1]
```diff
@@ -89,6 +89,10 @@
       {
         "Name": "JAVA_OPTS",
         "Value": "-Dservice.name=policy ${jvm_options} -XX:+HeapDumpOnOutOfMemoryError -Dnetworkaddress.cache.ttl=5 -Dnetworkaddress.cache.negative.ttl=5"
+      },
+      {
+        "Name": "LOG_LEVEL",
+        "Value": "${service_log_level}"
```
Comment (adityapahuja) [inline terraform/modules/hub/files/tasks/hub-policy.json:95]: Can we add the definition of service_log_level variable to  terraform/modules/hub/variables.tf.

Comment (adityapahuja) [reply terraform/modules/hub/files/tasks/hub-policy.json:95]: Can we add service_log_level to vars in template_file in policy, saml-engine, etc. For example, 

```
data "template_file" "policy_task_def" {
  template = file("${path.module}/files/tasks/hub-policy.json")

  vars = {
    ...
    service_log_level                   = var.service_log_level
  }
}
```

Comment (idrop) [reply terraform/modules/hub/files/tasks/hub-policy.json:95]: Ok, thanks. Terraform..

File: terraform/modules/hub/hub_saml_soap_proxy.tf [hunk 2]
```diff
@@ -73,6 +73,7 @@ data "template_file" "saml_soap_proxy_task_def" {
     certificates_config_cache_expiry = var.certificates_config_cache_expiry
     memory_hard_limit                = var.saml_soap_proxy_memory_hard_limit
     jvm_options                      = var.jvm_options
+    service_log_level                = var.service_log_level
```
Comment (adityapahuja) [inline terraform/modules/hub/hub_saml_soap_proxy.tf:76]: Can we update the task definition JSON file to pick up this variable `service_log_level `?

File: terraform/modules/hub/hub_saml_proxy.tf [hunk 3]
```diff
@@ -73,6 +73,7 @@ data "template_file" "saml_proxy_task_def" {
     certificates_config_cache_expiry = var.certificates_config_cache_expiry
     jvm_options                      = var.jvm_options
     memory_hard_limit                = var.saml_proxy_memory_hard_limit
+    service_log_level                = var.service_log_level
```
Comment (adityapahuja) [inline terraform/modules/hub/hub_saml_proxy.tf:76]: Can we update the task definition JSON file to pick up this variable `service_log_level `?

File: terraform/modules/hub/hub_saml_engine.tf [hunk 4]
```diff
@@ -57,6 +57,7 @@ data "template_file" "saml_engine_task_def" {
     certificates_config_cache_expiry = var.certificates_config_cache_expiry
     memory_hard_limit                = var.saml_engine_memory_hard_limit
     jvm_options                      = var.jvm_options
+    service_log_level                = var.service_log_level
```
Comment (adityapahuja) [inline terraform/modules/hub/hub_saml_engine.tf:60]: Can we update the task definition JSON file to pick up this variable `service_log_level `?

File: terraform/modules/hub/hub_config.tf [hunk 5]
```diff
@@ -89,6 +89,7 @@ data "template_file" "config_task_def" {
     metadata_object_key      = local.metadata_object_key
     memory_hard_limit        = var.config_memory_hard_limit
     jvm_options              = var.jvm_options
+    service_log_level        = var.service_log_level
```
Comment (adityapahuja) [inline terraform/modules/hub/hub_config.tf:92]: Can we update the task definition JSON file to pick up this variable `service_log_level `?

**Pull request merged**